shared/package.json

```diff
@@ -157,7 +157,7 @@
     "react-native-camera": "1.4.3",
     "react-native-contacts": "2.2.4",
     "react-native-fast-image": "5.1.1",
-    "react-native-image-picker": "0.27.1",
+    "react-native-image-picker": "git://github.com/keybase/react-native-image-picker#android-permissions-fix",
```

Comment:
this will be changed to v0.27.1-with-keybase-fixes once https://github.com/keybase/react-native-image-picker/pull/1 is merged